src/Poly.jl

```diff
@@ -1479,11 +1479,6 @@ function Base.divrem(f::PolyRingElem{T}, g::PolyRingElem{T}) where T <: RingElem
    return q, f
 end
 
-function Base.div(f::PolyRingElem{T}, g::PolyRingElem{T}) where T <: RingElement
-   q, r = divrem(f, g)
-   return q
-end
```

Comment:
redundant due to
https://github.com/Nemocas/AbstractAlgebra.jl/blob/c70d3b6900e4352ccffc22ff14b1f7b18516d21f/src/algorithms/GenericFunctions.jl#L77-L79